src/algorithm/arc/toPointsSvg.js

```diff
@@ -0,0 +1,113 @@
+const quantizeForSpace = require('../../math/utils/quantizeForSpace')
+const vec2 = require('../../math/vec2')
+
+/**
+ * Construct a line segment approximation of an arc, represented as an array of points.
+ * This implementation follows the SVG arc specs. For the details see
+ * http://www.w3.org/TR/SVG/paths.html#PathDataEllipticalArcCommands
+ * @param {Object} [options] - options for construction
+ * @param {Number} [options.xRadius=0] - X radius of arc.
+ * @param {Number} [options.yRadius=0] - Y radius of arc.
+ * @param {Number} [options.xaxisrotation=0] -  rotation (in degrees) of the X axis of the arc with respect to the X axis of the coordinate system
+ * @param {Number} [options.resolution=defaultResolution2D] - number of sides per 360 rotation
+ * @param {Boolean} [options.clockwise=false] - draw an arc clockwise with respect to the center point
+ * @param {Boolean} [options.large=false] - draw an arc longer than 180 degrees
+ * @param {vec2} startPoint - start of arc
+ * @param {vec2} endPoint - end of arc
+ * @returns {Array[vec2]} a point array.
+ *
+ * @example
+ * arc.toPointsSvg({ xRadius: 15, yRadius: -19.6875, xAxisRotation: 0, clockwise: false, large: false }, [0, 0], [12.5, -22.96875])
+ */
+const toPoints = ({ resolution = 32, xRadius = 0, yRadius = 0, xAxisRotation = 0, clockwise = false, large = false }, startPoint, endPoint) => {
```

Comment:
Hmm... I’m not good at names, but we can probably do better then toPoints(). I don’t like long names but this seems a lot better... generatePointsSVG()

And of course, the export and file name should match.

Reply:
Yes.

I think something like 'buildSvgArc' would be a better name.
Then 'buildCircleArc' can be put in alongside it, and so on.

What do you think?